lib/Service/Appointments/TimezoneGenerator.php

```diff
@@ -69,9 +61,15 @@ public function generateVtimezone(string $timezone, int $from, int $to): ?VTimeZ
 			}
 
 			if ($component) {
-				$date = new \DateTime($trans['time']);
-				$offset = $trans['offset'] / 3600;
-
+				if ($i === 0) {
+					$date = new \DateTime('19700101T000000');
```

Comment:
Feels a bit weird to use `1970-01-01` here.

Using `$trans['time']` would also work but it can stay as is.

Reply:
Yes, I know it looks odd, but this is how all the other software I have tested handles this, so I thought it would be best stay with the follow. The start year an date does not really matter for static time zones.

Reply:
Makes sense!